msgraph/accesspackage.go

```diff
@@ -17,7 +17,7 @@ type AccessPackageClient struct {
 
 func NewAccessPackageClient(tenantId string) *AccessPackageClient {
```

Comment:
This can be removed since it's no longer used?

```suggestion
func NewAccessPackageClient() *AccessPackageClient {
```